src/index.js

```diff
@@ -1,10 +1,17 @@
 import './style.css';
 import ICON from './assets/img/logo.png';
+import gamesApi from './modules/GamesAPI.js';
+import renderGame from './components/GameUI.js';
 
 window.addEventListener('load', () => {
   const logoContainer = document.querySelector('.logo-container');
   const logoIcon = new Image();
   logoIcon.src = ICON;
-
   logoContainer.appendChild(logoIcon);
+
+  gamesApi.games.forEach((game) => {
+    if (game.id < 10) {
+      renderGame(game);
+    }
+  });
```

Comment:
It seems gamesApi.games property of the class returns zero instead of our games available. This is probably because of the fact that the async function getGames() in the class hasn't yet returned it values yet. Could be improved by ensuring we check whether the array has values before rendering or to ensure the getGames() function has returned completely before letting the renderGame populate the UI